pkg/apis/core/validation/validation_test.go

```diff
@@ -7405,6 +7405,15 @@ func TestValidatePod(t *testing.T) {
 							},
 							TopologyKey: "zone",
 							Namespaces:  []string{"ns"},
+							NamespaceSelector: &metav1.LabelSelector{
+								MatchExpressions: []metav1.LabelSelectorRequirement{
+									{
+										Key:      "key",
+										Operator: metav1.LabelSelectorOpIn,
+										Values:   []string{"value1", "value2"},
+									},
+								},
+							},
```

Comment:
As mentioned in previous comment, can we add a test verifying matchExpressions[0].operator == Exist, while also having non-nil values. And let's get to a consensus what's the expected validator error(s).

Reply:
now that the validation is flipped, the invalid test case you are suggesting isn't different from the second one below that validates that the operator must be In